### PR TITLE
feat(RequestManager): create RequestManager

### PR DESCRIPTION
**Pull request by Hazmi35**
I'm trying to find a way to get the version and name from package.json without adding package.json to the dist folder, any ideas?

### CODE REVIEW

File: src/utils/RequestManager.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import got from "got";
+// import { version, name } from "../../package.json";
+
+export const RequestManager = got.extend({
+    prefixUrl: "https://discord.com/api/v8/",
+    responseType: "json"
+    // headers: { // NOTE: I'm trying to find a way to get version and name from package.json without adding package.json to dist folder
```
Comment (KurokuTetsuya) [inline src/utils/RequestManager.ts:7]: `process.cwd()` doesn't help?

Comment (Hazmi35) [reply src/utils/RequestManager.ts:7]: Hmm, then we need to export a function instead of variables

**Comment by Hazmi35** [comment]
Should we use node-fetch or stay with got?

**Comment by Waxaranai** [comment]
> Should we use node-fetch or stay with got?

Hmm... should we need to compare it first?

**Comment by Hazmi35** [comment]
Ready for review

**Review by KurokuTetsuya — APPROVED** [review]
Make it quick! We need another progress as fast as possible!

**Review by Waxaranai — APPROVED** [review]

**Comment by Hazmi35** [comment]
2 Reviewers is here, merging to the main branch

**Pull request merged**